google_sheets.gemspec

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require "google_sheets/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "google_sheets"
+  spec.version       = GoogleSheets::VERSION
+  spec.authors       = ["Jos Koetsier"]
```

Comment:
Can you use these values here?

https://github.com/alphagov/govuk_schemas/blob/master/govuk_schemas.gemspec#L9-L10